src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -2288,6 +2288,58 @@ public static IUIElement WaitForTabElement(this IApp app, string tabName)
 			return app.WaitForElementTillPageNavigationSettled(tabName);
 		}
 
+		/// <summary>
+		/// Performs platform-specific context actions (e.g., long press, swipe, or touch-and-hold) on a specified element in the app.
+		/// - On Android, it performs a long press gesture on the element.
+		/// - On Windows, it simulates a right-click (touch-and-hold) on the element.
+		/// - On iOS, it performs a swipe from right to left on the element.
+		/// - On Catalyst, it performs a scroll from right to left on the element.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="element">The element on which to perform the context action.</param>
+		public static void ActivateContextMenu(this IApp app, string element)
```

Comment:
LGTM. Here in this PR https://github.com/dotnet/maui/pull/26204 we have actions with the same, although using internal commands from each native driver. That is, in Android a long press is done etc.
Although the result is the same, in case something changes by updating Appium and something becomes deprecated, I think it is better to have a class per platform and to use each driver directly.